sycl/doc/UsersManual.md

```diff
@@ -290,6 +290,33 @@ and not recommended to use in production environment.
 
     NOTE: This flag is currently only supported with the CUDA and HIP targets.
 
+
+**`-f[no-]sycl-esimd-force-stateless-mem`** [EXPERIMENTAL]
+
+    Enforces stateless memory access and enables the automatic conversion of
+    "stateful" memory access via SYCL accessors to "stateless" within ESIMD
+    (Explicit SIMD) kernels.
+
+    -fsycl-esimd-force-stateless-mem disables the intrinsics and methods
+    accepting SYCL accessors or "surface-index" which cannot be automatically
+    converted to their "stateless" equivalents.
+
+    The stateless memory access enforcement is disabled by default.
+
+    -fno-sycl-esimd-force-stateless-mem is used to tell compiler not to
+    enforce usage of stateless memory accesses.
```

Comment:
```suggestion
    -fno-sycl-esimd-force-stateless-mem is used to tell compiler not to
    enforce usage of stateless memory accesses. This is default behavior.
```
nit: looks a bit clearer to me this way.

Reply:
Ok, Thank you. Fixed here: 11d9c6d5f6e687f1a1f438143b7a59ca4ca3e06a